apps/admin-gui/src/app/shared/components/dialogs/edit-attribute-definition-dialog/edit-attribute-definition-dialog.component.html

```diff
@@ -0,0 +1,183 @@
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeSelfPublic">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeSelfVo">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeVo">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeGroup">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeFacility">-->
+          <!--        </mat-checkbox>-->
+          <!--      </div>-->
+
+          <div class="d-flex flex-row mt-2">
+            <p class="ml-5 pl-4">{{'DIALOGS.CREATE_ATTRIBUTE_DEFINITION.READ' | translate}}</p>
+            <p class="ml-4">{{'DIALOGS.CREATE_ATTRIBUTE_DEFINITION.WRITE' | translate}}</p>
+          </div>
+
+          <div class="d-flex flex-row">
+            <p class="">{{'DIALOGS.CREATE_ATTRIBUTE_DEFINITION.SELF' | translate}}</p>
+            <mat-checkbox class="ml-5 pl-3" [(ngModel)]="readSelf">
+            </mat-checkbox>
+            <mat-checkbox class="ml-5" [(ngModel)]="writeSelf">
+            </mat-checkbox>
+          </div>
+
+          <div class="d-flex flex-row">
+            <p class="">{{'DIALOGS.CREATE_ATTRIBUTE_DEFINITION.SELF_PUBLIC' | translate}}</p>
+            <mat-checkbox class="ml-3" [(ngModel)]="readSelfPublic">
+            </mat-checkbox>
+            <mat-checkbox class="ml-5" [(ngModel)]="writeSelfPublic">
+            </mat-checkbox>
+          </div>
+
+          <div class="d-flex flex-row">
+            <p class="">{{'DIALOGS.CREATE_ATTRIBUTE_DEFINITION.SELF_VO' | translate}}</p>
+            <mat-checkbox class="ml-4 pl-3" [(ngModel)]="readSelfVo">
+            </mat-checkbox>
+            <mat-checkbox class="ml-5" [(ngModel)]="writeSelfVo">
+            </mat-checkbox>
+          </div>
+
+          <div class="d-flex flex-row">
+            <p class="">{{'DIALOGS.CREATE_ATTRIBUTE_DEFINITION.VO' | translate}}</p>
+            <mat-checkbox class="ml-5 pl-4  " [(ngModel)]="readVo">
+            </mat-checkbox>
+            <mat-checkbox class="ml-5" [(ngModel)]="writeVo">
+            </mat-checkbox>
+          </div>
+
+          <div class="d-flex flex-row">
+            <p class="">{{'DIALOGS.CREATE_ATTRIBUTE_DEFINITION.GROUP' | translate}}</p>
+            <mat-checkbox class="ml-4 pl-4" [(ngModel)]="readGroup">
+            </mat-checkbox>
+            <mat-checkbox class="ml-5" [(ngModel)]="writeGroup">
+            </mat-checkbox>
+          </div>
+
+          <div class="d-flex flex-row">
+            <p class="">{{'DIALOGS.CREATE_ATTRIBUTE_DEFINITION.FACILITY' | translate}}</p>
+            <mat-checkbox class="ml-4 pl-3" [(ngModel)]="readFacility">
+            </mat-checkbox>
+            <mat-checkbox class="ml-5" [(ngModel)]="writeFacility">
+            </mat-checkbox>
+          </div>
+
+          <div class="mt-3">
```

Comment:
It would be nice to have some label saying that the table represents services that use this attribute. Otherwise, it is not clear.

Reply:
Added 'Dependent services:'